sql/moz-fx-data-shared-prod/telemetry_derived/sponsored_tiles_clients_daily_v1/query.sql

```diff
@@ -154,6 +154,20 @@ ios_data AS (
     1,
     2
 ),
+overactive_android_clients AS (
+  -- Find client_ids with over 150 000 pings in a day,
+  -- which could cause errors in the next step due to aggregation overflows.
```

Comment:
Isn't this 150 000 events per day across all the pings?  Maybe this should be a little higher than the ping limit or use `fenix.events`

Reply:
Good point. I increased it to 3mil, which looking at the ping counts per client seems reasonably high enough